python/tk_multi_publish2/publish_tree_widget/publish_tree_widget.py

```diff
@@ -448,12 +469,8 @@ def select_first_item(self):
 
         else:
             # summary hidden. select first item instead.
-            first_item = None
-            for context_index in range(1, self.topLevelItemCount()):
-                context_item = self.topLevelItem(context_index)
-                for child_index in range(context_item.childCount()):
-                    first_item = context_item.child(child_index)
-                    break
+            # get the first item from the root_items generator.
+            first_item = next(self.root_items(), None)
```

Comment:
There should be no change here in behaviour, this is simply to make use of the tree widget's new `root_items` generator.